### ISSUE TITLE
Add an initial Launchpad task list to the Reader

### ISSUE THREAD

**Issue by markbiek** [Automattic/wp-calypso#84377]
As a first step towards improving the Reader onboarding experience, let's try adding a Launchpad task list to the Reader. We want to make sure we have good tracking on the task list so we can measure if people are interacting with the list _and_ measure if it seems to increase Reader engagement.

Important: Make sure the task list is dismissable!

Possible task to include in the list:

- Follow a blog on WordPress.com
- Follow a blog NOT on WordPress.com. This is a good one because it helps surface the fact that the Reader can be used as an RSS feed aggregator.
- Like a post.
- Comment on a post.
- Follow a tag.
- Search for something
- Explore first posts (or a more user-friendly label, like "Explore first posts from new sites" )
- Explore Discover and/or the tags to find some content -- this might need to be first on the list

More information on a different, walk-through type approach can be found here:

- pe7F0s-1tY-p2
- pe7F0s-1r4-p2

cc @Automattic/serenity

**Comment by daledupreez** [comment]
Also cc'ing @Automattic/loop for visibility.

As a related comment, even if users aren't interacting with the specific tasks, we're still showing them things they can do with the Reader, which is more than we're doing at the moment.

I've also added two possible tasks that we discussed while brainstorming:
 - Explore first posts
 - Explore Discover/tags to find some content

**Comment by roo2** [comment]
This sounds a little bit similar to the Reader onboarding idea, where we would present the user with a list of tags to follow, shown here at the dotcom hackathon https://video.wordpress.com/v/om2vhslG?at=6190. I'm not sure if this has been officially mocked up yet but I think it's a good idea and it sounds similar cc @osullivanchris

**Comment by osullivanchris** [comment]
Just to avoid a mistake we've made in the mobile app. We have an onboarding tour but it feels a bit like a chore. We send people around to tap on a bunch of things. Its more centred on "look at our UI and how it works" than what the user goals are. Just playing around with the above ideas and roughly categorising by things the user might want to achieve.

**Find inspiring things to read:**
- Discover
- Check out first posts from other bloggers

**Personalise the Reader to things you're interested in:**
- Follow tags based on your interests
- Search for blogs/posts (but maybe this is not strong enough today so I might omit it)
- If you have a site that you like that isn't on WordPress, did you know you can follow via RSS
- See these things showing up in your Following feed
- Save posts to read later

**Interact with other people to make connections and bring traffic to your own blog:**
- Like and comment on other posts
- Follow blogs you're interested in

As I write this out I wonder should it be a checklist? It feels more like a knowledge base. Some of them you might want to revisit. Some you can't really "complete" as such. They might work if they were static guidance.

I also think (biased) that we should suggest the mobile app for reading. 
- Read on the go
- Dark mode
- Save posts to read later or offline
- Customise the fonts/colors (not built yet :) )

**Comment by markbiek** [comment]
> As I write this out I wonder should it be a checklist? It feels more like a knowledge base. Some of them you might want to revisit. Some you can't really "complete" as such. They might work if they were static guidance.

That's actually exactly how this will work if we use the Launchpad in the Reader.

Imagine something like this, but dismissable:
<img width="707" alt="CleanShot 2023-11-27 at 07 32 42@2x" src="https://github.com/Automattic/wp-calypso/assets/917632/a34be07c-12c6-4fb2-af18-60ec9e70fddd">

**Comment by eoigal** [comment]
I like the idea of a launchpad, we discussed something like this on our team day at DM;

Couple of things we'd like to get our Readers to do if they haven't already...

1. Choose from a selection of tags that outlines their interests and what content they'd like to see on Reader
2. Choose for a static list of known quality RSS feeds similar to other Reader apps (this will help fill the Reader with quality content for early users).

These things don't have to be done via a launchpad. We were discussing using a modal/dialog to prompt users to go through some select steps... perhaps we can do both?

**Comment by Addison-Stavlo** [comment]
A launchpad feels very rigid and overkill for the context of the reader IMO. A task list to set up a specific type of site or reach certain site goals is one thing, but a task list to have a user reach arbitrary goals with respect to interacting with the reader feels quite odd. The former is expected to be work that guidance will help achieve, the latter would feel noisy and make the reader feel more like a chore.

The main things we need to do for onboarding in the reader are getting the user to follow some tags of interest, some sites of interest, and maybe some external RSS feeds of interest. This can all be done in-place in a quick interaction mechanism that does not require the rigidity of tracking tasks and completions. We just need a quick welcome experience.

[This is roughly the idea we were looking at](pe7F0s-1nN-p2#comment-1768) for an i1 of reader onboarding that should drastically improve the relevance of discover and following feeds within a few seconds of entering the reader.

**Comment by Addison-Stavlo** [comment]
At first glance launchpad feels very rigid and overkill for the context of the reader. A task list to set up a specific type of site or reach certain site goals is one thing, but a task list to have a user reach arbitrary goals with respect to interacting with the reader feels quite odd. The former is expected to be work that guidance will help achieve, the latter would feel noisy and make the reader feel more like a chore **_if we aren't careful._**

If we go for something like this, it would be good to make it more quirky and gamified (more like achievements to celebrate instead of requirements to action?). In a non-intrusive location that is still relevant to the user? 🤔 

The main (immediate) things we need to do for onboarding in the reader are getting the user to follow some tags of interest, some sites of interest, and maybe some external RSS feeds of interest. This can all be done in-place in a quick interaction mechanism that does not require the rigidity of tracking tasks and completions. We need a quick welcome experience that can surface this to users. This is roughly the idea we were looking at (pe7F0s-1nN-p2#comment-1768) for an i1 of reader onboarding that should drastically improve the relevance of discover and following feeds within a few seconds of entering the reader.

I imagine some sort of gamified task-list could still fit in well after this welcome perhaps? 🤔

**Comment by daledupreez** [comment]
> This can all be done in-place in a quick interaction mechanism that does not require the rigidity of tracking tasks and completions. We just need a quick welcome experience.

I don't agree that the launchpad is overkill, especially in terms of effort. The work to create a new set of tasks for the launchpad is really straightforward, as it's mostly back end work to add URLs, translated strings, and optionally some quick checks for completion (though we could simply mark each task as complete when the user clicks on them - that needs them to be added to a client-side constant). My concern is that while a custom modal may be "easy", it's also one more thing we need to maintain and keep up-to-date.

It may be that we want the more customised welcome in the long run, but it may be faster to start with a launchpad and see how that performs. Also, we definitely need an easy way to dismiss the launchpad for this context so it can be ignored.

**Comment by Addison-Stavlo** [comment]
👋  Hi @daledupreez, thanks for the quick response. I was updating my reply a couple times there as I thought about it more so it may read a little different now.

> I don't agree that the launchpad is overkill, especially in terms of effort. 

I don't think overkill in terms of effort, more in terms of interaction. Sending the user back and forth to achieve the individual things, etc. Some of these initial concerns should be fast and streamlined (initial interests to follow and subscription recommendations based on that).

> My concern is that while a custom modal may be "easy", it's also one more thing we need to maintain and keep up-to-date.

What is the maintenance you would be concerned about here?  Note for "easy" i mean from the users perspective (not just development effort). Also note that the standalone reader app exploration, tumblr, and i think the mobile app now (?) all use patterns very similar to this already.

I wonder if there is sense in doing both at some level?

**Comment by Addison-Stavlo** [comment]
> I wonder if there is sense in doing both at some level?

Riffing on this more... the "initial interests to follow / suggested subscriptions welcome modal thingy" could even introduce the checklist to go on from there. And the checklist could continue from there with gamifying and guiding likes, comments, different management interfaces, and getting users more comfortable with the reader as a whole as well.

**Comment by Addison-Stavlo** [comment]
A couple other ideas for tasks in the list could be:
* update a subscription's settings (notifs, emails for posts and comments, daily vs. weekly, etc.)
* upload a gravatar image (if none is already set) - @simison mentioned this one previously and may have some other ideas too!

**Comment by osullivanchris** [comment]
@Addison-Stavlo your concerns ring true for me. My comment touched on similar points, that we have this pitfall in the app where it feels more like we're sending the user to do a bunch of chores than to help them with their goals.

I think having a full-view onboarding experience as you mention could be more compelling. And then when you see the 'unfinished' tasks somewhere in the UI afterwards, there's more of a connection to what you did initially. As we both said I think we also need to be very careful about what the tasks are. Following some tags, showing how to find a site, seem like necessities. As we go further and say 'go and like/comment on other blogs' that feels less like onboarding guidance. It could be more suitable for a gamification angle as you mentioned, like collecting badges for reading 10 posts, following 10 blogs, liking 10 posts...

**Comment by simison** [comment]
> could be more suitable for a gamification angle as you mentioned, like collecting badges for reading 10 posts, following 10 blogs, liking 10 posts...

Nice idea. Might require different design or copy, but could otherwise be fun addition. Haven't seen that elsewhere!

>upload a gravatar image (if none is already set) - @simison mentioned this one previously and may have some other ideas too!

Yeah, updating profile in general. See also how it looks to comment/like/write now without profile name & image, and see if those places offer ways to edit it? There's probably lots to improve there even without launchpad.

If they're a publisher, finding sites to recommend will be a good nudge once we have those mechanisms in place. Now we have only the manual blogroll block.
